colorbleed/plugins/global/publish/submit_publish_job.py

```diff
@@ -155,15 +159,17 @@ def process(self, instance):
         # This assumes the output files start with subset name and ends with
         # a file extension.
         if "ext" in instance.data:
-            ext = re.escape(instance.data["ext"])
+            ext = instance.data["ext"].strip(".")
```

Comment:
I have chosen to do this because I explicitly added `\.{ext}` to the regex string plus a double escape looked a bit odd.
What do you mean with "stability"? Are you referring to possible double extensions like `super.pig`?

Reply:
I have chosen to do this because I explicitly added `\.{ext}` to the regex string plus a double escape looked a bit odd.
What do you mean with "stability"? Are you referring to possible double extensions like `super.pig`?